File: src/main/java/ru/dragonestia/apocalypse/item/PlasticItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class PlasticItem extends Item implements AdvItem {

    public PlasticItem(Integer meta, int count) {
        super(ApocalypseID.PLASTIC, meta, count, "Plastic");
    }

    @Override
    public String getRuName() {
        return "Пластик";
    }

    @Override
    public String getIcon() {
        return "textures/items/plastic";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/PlasticItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class PlasticItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.PLASTIC, 0, count, "Plastic"), AdvItem {

    override fun getRuName(): String = "Пластик"

    override fun getIcon(): String = "textures/items/plastic"

}